### ISSUE TITLE
MTV Upload Failing

### ISSUE THREAD

**Issue by saddle-gaudier-06** [Audionut/Upload-Assistant#47]
When generating the torrent for MTV, the script says 

"Uploading to MTV
Successfully logged in to MTV
Searching for existing torrents on site...
No dupes found
Images are already hosted on an approved image host. Skipping re-upload." 

However, once the torrent is uploaded the script errors out saying, "Upload Failed, Doesnt look like you are logged in"

I've filled my username, password, and otp_uri for MFA in the config file.

Any ideas of a potential solution?

**Comment by Audionut** [comment]
No issues here. It's not clear exactly what is failing though. The torrent uploaded, but didn't get added to client, or?

Possible that the site decided to logout after the dupe check I guess. Can you run another attempt and post output?

**Comment by Audionut** [comment]
I've managed to trigger the bug, let me look into it.

**Comment by Audionut** [comment]
Yeah, someone pinged me somewhere else about this, and when I had the time to check, it worked for me also. Blastered thing.

I was going through some old issues and came across a similar issue https://github.com/L4GSP1KE/Upload-Assistant/issues/310
Manual cookiefiles is messy and not something I'm keen to entertain.

Anyway, this is at the top of my list of things to get fixed properly, if possible.

**Comment by Kha-kis** [comment]
I had this happen when I had above 25 unchecked torrents and worked once under that limit. Did you have the same scenario?

**Comment by Audionut** [comment]
I was getting it to trigger on a torrent that needed MTV specific rehash. I hadnt been uploading much, so doubt i was near 25 unchecked torrents. Given that its not consistent, I suspect some site side issue is at least partly responsible.

**Comment by Audionut** [comment]
Had some help to debug this with someone who found a method to consistently trigger the bug.

It's a site bug: https://www.morethantv.me/forum/thread/3338?

**Comment by Audionut** [comment]
This should be pretty well covered in the feedback in program.
